src/FSharpVSPowerTools.Core/LanguageService.fs

```diff
@@ -354,10 +352,7 @@ type LanguageService (?fileSystem: IFileSystem) =
           | Some results -> return results
           | None -> 
               debug "Parsing: Trigger parse (fileName=%s)" fileName
-              let! results = parseAndCheckFileInProject(fileName, src, opts)
-              debug "Worker: Starting background compilations"
-              checkerInstance.StartBackgroundCompile opts
```

Comment:
@vasily-kirichenko I'm not sure we should be removing this until we've got to the of CPU and memory usage issues.  

VFT uses a background compile - for example, the background compile for a project is kicked off whenever you switch to a file in that project.  This of course uses CPU when editing and memory to store the results - but it means that lots of requests (e.g. intellisense and symbols) are very quick. 

In principle it's reasonable for VFPT to be doing the same - if you don't, then you will have to do more compute when doing things like rename refactoring .  The question I'd relaly like to get to the bottom of before making fairly large behavioural changes like this is why adding the VFPT/FCS background build uses so much memory, and also whether particular VFPT/FCS features are causing unnecessary CPU load.

So I'd recommend holding off on this change just at the moment.